account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -69,15 +72,35 @@ def import_file(self):
 
     @api.model
     def _import_file(self, data_file):
-        """ Create bank statement(s) from file
-        """
+        """ Create bank statement(s) from file."""
         # The appropriate implementation module returns the required data
-        currency_code, account_number, stmts_vals = self._parse_file(data_file)
-        # Check raw data
-        self._check_parsed_data(stmts_vals)
+        statement_ids = []
+        notifications = []
+        statements = self._parse_file(data_file)
```

Comment:
@NL66278  @hbrunn I did not pay enough attention to this point. This change breaks the current API and therefore the parsers developed in some localisations (https://github.com/acsone/l10n-belgium/blob/8.0-add-account_bank_statement_import_coda/account_bank_statement_import_coda/wizard/account_bank_statement_import_coda.py#L62).
To keep the backward compatibility you can may be define a new function implemented by some specifics parsers `_parse_multi_smts_file` and refactor the code as
```python
    ....
    def __parse_file(self, data_file):
        return False

    def _parse_multi_stmt_file(data_file):
        return []
    ....
    @api.model
    def _import_file(self, data_file):
        ....
        statements = []
        stmt_vals = self._parse_file(data_file)
        if stmt_vals:
            statements = [stmt_vals]
        statement.extend(self._parse_multi_stmt_file(data_file))
        ...
```